apps/fullybinarytime/fullybinarytime.star

```diff
@@ -126,7 +126,7 @@ def make_frame(elapsed):
 def make_animation(timezone):
     # Key input is how long has passed since the start of the day.
     now = time.now().in_location(timezone)
-    midnight = time.time(year = now.year, month = now.month, day = now.day).in_location(timezone)
```

Comment:
This would create midnight GMT and then shift it into the other timezone (e.g. 8pm in New York). What we actually want is midnight in the timezone (12pm in New York)